src/test/java/seedu/address/testutil/ModelStub.java

```diff
@@ -42,11 +42,21 @@ public Path getAddressBookFilePath() {
         throw new AssertionError("This method should not be called.");
     }
 
+    @Override
+    public Path getEventBookFilePath() {
+        return null;
```

Comment:
throw new AssertionError("This method should not be called.");
may i ask why u used `return null` instead of the above? if this would cause problem can let us know in the group

Reply:
I think I chatgpt this. Sorry, I should check.